realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/mongodb/sync/SyncedRealmTests.kt

```diff
@@ -508,6 +509,107 @@ class SyncedRealmTests {
         }
     }
 
+    @Test
+    fun allTypes_roundTrip() {
+        val user1: User = createNewUser()
+        val config1: SyncConfiguration = createDefaultConfig(user1, partitionValue)
+
+        val primaryKeyValue = ObjectId()
+        val expectedRealmInteger = 100.toLong()
+        val expectedString = "hello world"
+        val expectedLong = 10.toLong()
+        val expectedDouble = 10.0
```

Comment:
Missing float test

Reply:
Also, we should probably try to store an value that doesn't accidently becomes correct. I think normally we use `1.23F` and `1.234D`

Reply:
Ah wait. Float isn't supported by Sync, right?

Reply:
Yes, it isn't. We have a specific object for testing that float fails with sync: SyncAllTypesWithFloat